source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -31,42 +39,258 @@ export default class PinCode extends Component<Props> {
   inputsRef = [];
   focusKey = 0;
   add = false;
+  fromBackspace = false;
+
+  state = {
+    isBackSpace: false,
+    focusKeyChanged: false,
+    focusIsUpdated: false,
+  };
+
+  hasInputNewValue = (inputNewValue: string, key: number) => {
+    const { value } = this.props;
+    const valueIsEmpty =
+      !value[key] || value[key] === '' || inputNewValue === '';
+    const valueHasChanged =
+      inputNewValue &&
+      inputNewValue.length === 1 &&
```

Comment:
just add check if exist and not empty e.g. `inputNewValue && ...`

Reply:
Fixed